tests/z80benchtests/z80benchtests.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.1" />
```

Comment:
Don't have a huge amount of C# exp, so best practices/defacto standards elude me, not sure if Microsofts default of MSTest is the way to go, but seems to work well enough